src/main/drivers/timer.c

```diff
@@ -50,13 +50,13 @@
 
 #define TIM_IT_CCx(ch) (TIM_IT_CC1 << ((ch) / 4))
 
-typedef struct timerConfig_s {
+typedef struct timerCallbackConfig_s {
```

Comment:
Fair enough. This can easily be renamed back now in any case, as I ended up calling it timerChannelConfig